index.ts

```diff
@@ -166,261 +219,154 @@ async function init() {
-          },
-          initial: true,
-          active: language.defaultToggleOptions.active,
-          inactive: language.defaultToggleOptions.inactive,
-        },
-        {
-          name: 'overwriteChecker',
-          type: (prev, values) => {
-            if (values.shouldOverwrite === false) {
-              throw new Error(red('✖') + ` ${language.errors.operationCancelled}`)
-            }
-            return null
-          },
-        },
-        {
-          name: 'packageName',
-          type: () => (isValidPackageName(targetDir) ? null : 'text'),
-          message: language.packageName.message,
-          initial: () => toValidPackageName(targetDir),
-          validate: (dir) => isValidPackageName(dir) || language.packageName.invalidMessage,
-        },
-        {
-          name: 'needsTypeScript',
-          type: () => (isFeatureFlagsUsed ? null : 'toggle'),
-          message: language.needsTypeScript.message,
-          initial: false,
-          active: language.defaultToggleOptions.active,
-          inactive: language.defaultToggleOptions.inactive,
-        },
-        {
-          name: 'needsJsx',
-          type: () => (isFeatureFlagsUsed ? null : 'toggle'),
-          message: language.needsJsx.message,
-          initial: false,
-          active: language.defaultToggleOptions.active,
-          inactive: language.defaultToggleOptions.inactive,
-        },
-        {
-          name: 'needsRouter',
-          type: () => (isFeatureFlagsUsed ? null : 'toggle'),
-          message: language.needsRouter.message,
-          initial: false,
-          active: language.defaultToggleOptions.active,
-          inactive: language.defaultToggleOptions.inactive,
-        },
-        {
-          name: 'needsPinia',
-          type: () => (isFeatureFlagsUsed ? null : 'toggle'),
-          message: language.needsPinia.message,
-          initial: false,
-          active: language.defaultToggleOptions.active,
-          inactive: language.defaultToggleOptions.inactive,
-        },
-        {
-          name: 'needsVitest',
-          type: () => (isFeatureFlagsUsed ? null : 'toggle'),
-          message: language.needsVitest.message,
-          initial: false,
-          active: language.defaultToggleOptions.active,
-          inactive: language.defaultToggleOptions.inactive,
-        },
-        {
-          name: 'needsE2eTesting',
-          type: () => (isFeatureFlagsUsed ? null : 'select'),
-          hint: language.needsE2eTesting.hint,
-          message: language.needsE2eTesting.message,
-          initial: 0,
-          choices: (prev, answers) => [
+
+  if (!targetDir) {
+    targetDir =
+      result.projectName =
+      result.packageName =
+        await unwrapPrompt(
+          text({
+            message: language.projectName.message,
+            placeholder: defaultProjectName,
+            validate: (value) => (value.trim().length > 0 ? undefined : 'Should not be empty'),
```

Comment:
nit: this should be translated as well no? French translation: `Ne doit pas être vide`.